gammapy/data/simulate.py

```diff
@@ -13,14 +13,14 @@ class ObservationsEventsSampler(parallel.ParallelMixin):
         Arguments passed to `~gammapy.datasets.MapDatasetEventSampler`.
     dataset_kwargs : dict, optional
         Arguments passed to `~gammapy.datasets.create_map_dataset_from_observation()`.
-    outdir : str, Path
-        path of the output files created. Default is "./simulated_data/".
-        If None a list of `~gammapy.data.Observation` is returned.
-    overwrite : bool
-        Overwrite the output files or not
+    outdir : str, optional
+        Path of the output files created. Default is "./simulated_data/".
+        If None, a list of `~gammapy.data.Observation` is returned.
+    overwrite : bool, optional
+        Overwrite existing file. Default is True.
     n_jobs : int, optional
         Number of processes to run in parallel.
-        Default is one, unless `~gammapy.utils.parallel.N_JOBS_DEFAULT` was modified.
+        Default is None, unless `~gammapy.utils.parallel.N_JOBS_DEFAULT` was modified.
```

Comment:
The default is one  as it is the default for N_JOBS_DEFAULT. Maybe you can find a better phrasing, but it should still mention one.